test/control-plane/services/dual-stack/manifests/echo_svc_dualstack.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
let's be consistent in the file names, let's change `echo-svc-dualstack` instead of `_`

Reply:
should I also fix the source of this, e.g. `test/k8s/manifests/echo_svc_dualstack.yaml` and the other files there?

Reply:
> should I also fix the source of this, e.g. `test/k8s/manifests/echo_svc_dualstack.yaml` and the other files there?
> 

Those will eventually disappear so there is no need to fix them.

> Note that I copied this and other manifests as slight tweaks were needed, and to me made sense to keep them with the specific test case.